applications/hotornot/doc/README.md

```diff
@@ -1,2 +1,95 @@
+# Hot or Not: *Call rating*
 
-# Hot or Not *Call rating*
+This application serves `rate_req` AMQP requests.
+
+Rates are stored in a system-wide "ratedeck" database.
+
+Each rate is a separate document:
+
+```json
+{
+    "_id": "f36cac061205f7a0f7fde6b4d3370bf9",
+    "_rev": "1-1b67fb507c69c35200178a9b1e0e6cfc",
+    "routes": [
+        "^\\+?1.+$"
+    ],
+    "weight": "1",
+    "rate_name": "outbound_US_1",
+    "pvt_type": "rate",
+    "prefix": "1",
+    "rate_cost": "1.27",
```

Comment:
Then maybe it should be in units? Or if it's USD it should say so, shouldn't it?